README.md

```diff
@@ -46,3 +46,12 @@ the [Telemetry Data Collection Notice](TELEMETRY.md)
 For issues relating to a specific devfile stack in this repository, please reach out to the devfile stack maintainer to determine where to open an issue.
 
 For issues relating to the hosted devfile registry service (<https://registry.devfile.io>), or devfile registries in general, please use the [devfile/api](https://github.com/devfile/api/) issue tracker for opening issues. Apply the `area/registry` label to registry issues for better visibility.
+
+## Badges
```

Comment:
@lmilbaum for this section, I like the idea of having a badge for renovate. I'm slightly concerned having a the `badges` section might be misleading for someone as renovate atm only updates the registry stack image versions and not all the dependencies used inside the registry repo.

I would suggest moving the badge:

```
[![Renovate][1]][2]
[1]: https://img.shields.io/badge/renovate-enabled-brightgreen?logo=renovate
[2]: https://renovatebot.com
```

Inside the [regigstry status](https://github.com/devfile/registry/blob/main/README.md#registry-status) section.

Reply:
Done